src/kontrol/kdist/cheatcodes.md

```diff
@@ -896,6 +897,7 @@ The `ECDSASign` function returns the signed data in [r,s,v] form, which we conve
          <k> #call_foundry SELECTOR ARGS => . ... </k>
          <output> _ => #sign(#range(ARGS, 32, 32),#range(ARGS,0,32)) </output>
       requires SELECTOR ==Int selector ( "sign(uint256,bytes32)" )
+      [preserves-definedness]
```

Comment:
In this rule, by definition of
```
rule #sign(BA1, BA2) => #parseByteStack(ECDSASignbytes(BA1, BA2)) [concrete]
```
and the totality of `ECDSASignbytes` (assuming its correct implementation), `#parseByteStack` should always be able to always unparse the result. With this in mind, could we mark `#sign` as total?